backend/benefit/applications/services/ahjo_integration.py

```diff
@@ -31,34 +26,27 @@
         "file_name": "Helsinki-lisä 2021 koontiliite yhteisöille mallipohja "
```

Comment:
I think "mallipohja" (=template) maybe shouldn't be part of the actual output file name, as that file is not a template

Reply:
Thanks, I'll update it